sdks/python/apache_beam/runners/interactive/pipeline_instrument_test.py

```diff
@@ -48,7 +50,7 @@ def setUp(self):
 
   def test_pcolls_to_pcoll_id(self):
     p = beam.Pipeline(interactive_runner.InteractiveRunner())
-    # pylint: disable=range-builtin-not-iterating
+# pylint: disable=range-builtin-not-iterating
```

Comment:
nit: the indent, probably caused by auto-formatter?

Reply:
Probably, thanks!